src/itest/kotlin/nl/info/zac/itest/HealthCheckRestServiceTest.kt

```diff
@@ -108,8 +139,8 @@ class HealthCheckRestServiceTest : BehaviorSpec({
                           "versiedatum": "$DATE_2023_09_21",
                           "vertrouwelijkheidaanduiding": "openbaar"
                         }
-                      }
-                    ]
+                      }            
+                    ]                    
```

Comment:
I guess detekt does not mind, but I guess we can remove the whitespaces at the end

Reply:
weird. Detekt should definitely mind..

Reply:
fixed